Libplanet/Net/Swarm.cs

```diff
@@ -1572,15 +1572,6 @@ CancellationToken cancellationToken
                         blockChain.Swap(synced, evaluateActions);
                     }
                 }
-
-                IStore store = BlockChain.Store;
-                foreach (Guid chainId in store.ListChainIds().ToList())
-                {
-                    if (!chainId.Equals(blockChain.Id))
```

Comment:
How about change this condition like below?
 
`!(chainId.Equals(blockChain.Id) || chainId.Equals(BlockChainId)`

Reply:
Do you mean `BlockChain.Id`?

Reply:
Right, `BlockChain.Id`. There was a typo.

Reply:
This was intended to remove chains that created by `FillBlocksAsync()`, so removing chains in it is better IMO.